packages/suite/src/components/suite/modals/ReviewTransaction/components/OutputList.tsx

```diff
@@ -165,6 +165,9 @@ const OutputList = ({
                         })}
                         {!precomposedTx.token && (
                             <OutputElement
+                                indicator={
+                                    <Indicator state={signedTx ? 'success' : undefined} size={16} />
```

Comment:
nit, optional: I didn't look  into details of the components, but I'd say it may be "cleaner" to pass no indicator at all rather than one with an undefined state. I.e.

```suggestion
                                    signedTx ? <Indicator state='success' size={16} /> : undefined
```

RBY

Reply:
indicator has 3 variants:
state undefined/default = grey dot
state active = orange dot
state success = green dot


proposed solution would render not dot at all :)

it's ok like this

Reply:
Allright. Thx.